pkg/scheduler/plugins/util/gangextender/gangextender.go

```diff
@@ -0,0 +1,42 @@
+/*
+ Copyright 2023 The Volcano Authors.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package gangextender
+
+import (
+	"k8s.io/klog/v2"
+	"volcano.sh/volcano/pkg/scheduler/api"
+)
+
+const (
+	sparkPodLabelRoleKey         = "spark-role"
+	sparkPodLabelRoleDriverValue = "driver"
+)
+
+func GangPreemptRejectExtender(preemptee *api.TaskInfo) bool {
+	sparkReject := sparkGangExtender(preemptee)
+	return sparkReject
+}
+
+func sparkGangExtender(preemptee *api.TaskInfo) bool {
```

Comment:
I think the function name is not meaningful. May let the function name has something relative with "driver role" is easy to understand.